gtk/Widget.custom

```diff
@@ -393,3 +393,14 @@ public void ModifyText (Gtk.StateType state)
 {
 	gtk_widget_modify_text_ptr (Handle, (int) state, IntPtr.Zero);
 }
+
+[DllImport("libgtk-win32-2.0-0.dll", CallingConvention = CallingConvention.Cdecl)]
+static extern void gtk_widget_size_request(IntPtr raw, ref Gtk.Requisition requisition);
+
+public Gtk.Requisition SizeRequest()
+{
+        Gtk.Application.AssertMainThread();
+        Gtk.Requisition requisition = Requisition;
```

Comment:
So, due to the nature of how marshalling works, we will never hit [this](https://github.com/bratsche/gtk/blob/xamarin-mac/gtk/gtkwidget.c#L4004-L4007). It seems native does not like sending over the actual widget's requisition.

Reply:
Hello, this fix solved our leak issue. Please, is it possible to merge this PR?
regards,

Reply:
Hello Therzok,
The solution is to replace the instanciation of Gtk.Requisition by the reference to the property Requisition stored in the class Gtk.Widget.
So basically, the correction is the following:

Before:
[DllImport("libgtk-win32-2.0-0.dll", CallingConvention = CallingConvention.Cdecl)]
static extern void gtk_widget_size_request(IntPtr raw, ref Gtk.Requisition requisition);
 
public Gtk.Requisition SizeRequest() {
      Gtk.Requisition requisition;
      Gtk.Application.AssertMainThread();
      **requisition = new Gtk.Requisition();**
```suggestion
        requisition = Requisition;
```
      gtk_widget_size_request(Handle, ref requisition);
       return requisition;
}